airgun/widgets.py

```diff
@@ -1267,6 +1287,96 @@ def read(self):
             "return ace.edit('{0}').getValue();".format(self.ace_edit_id))
 
 
+class Pagination(Widget):
+    """Represents Paginator widget that includes per page selector, First/Last/Next/Prev buttons
+    and current page index/overall amount of pages. Mainly used with Table widget.
+    """
+    ROOT = "//form[contains(@class, 'content-view-pf-pagination')]"
```

Comment:
Yes both has that locator, not sure about the relative as it will be used in table and it is not located under table.

Reply:
tried simply does not work

Reply:
ah, right, makes sense. By using absolute path we're risking to pick smth wrong, e.g. paginator from first inactive tab, but since it's used in table and is actually located outside of it, i don't think we have a choice.